### ISSUE TITLE
Include practice to use loop and loop_var

### ISSUE THREAD

**Issue by bontreger** [redhat-cop/automation-good-practices#54]
As per this ansible lint role, a recommendation to use loop and loop_var - https://ansible-lint.readthedocs.io/en/latest/default_rules/#no-loop-var-prefix

**Comment by ericzolf** [comment]
Good point. I don't think we need a specific rule though, it should just be highlighted as a special case of the generic rule to prefix vars with the role's name.

**Comment by ericzolf** [comment]
Do you create a PR for this?

**Comment by bontreger** [comment]
Not yet, I was brain-dumping as I was going through the ansible lint rules on a client's playbooks and this was a finding in lint that we haven't addressed
